ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
+                   toolbarColor:(UIColor * __tc_nonnull)toolbarColor
+{
+    [self tc_addToolbarWithButton:@"Next" toolbarColor:toolbarColor target:nextTextField action:@selector(becomeFirstResponder)];
+}
+
+- (void)tc_addToolbarWithButton:(NSString * __tc_nonnull)buttonTitle toolbarColor:(UIColor * __tc_nonnull)toolbarColor target:(id)target action:(SEL)selector
+{
+    UIBarButtonItem *button = [[UIBarButtonItem init] initWithTitle:buttonTitle style:UIBarButtonItemStylePlain target:target action:selector];
```

Comment:
How do you test this? The function returns void and it's a class function so I am not modifying an instance I can then test.